nextflow_schema.json

```diff
@@ -437,7 +437,7 @@
             "properties": {
                 "aligner": {
                     "type": "string",
-                    "default": "star",
+                    "default": "star_salmon",
                     "description": "Specifies the alignment algorithm to use - available options are 'star_salmon', or 'star'.",
```

Comment:
please add an `enum` entry for this parameter to improve validation and documentation